docs/common/registration.md

```diff
@@ -112,7 +112,7 @@ To register a new resource, follow the instructions below:
 1. Make changes with the [GitHub file editor](https://help.github.com/articles/editing-files-in-your-repository/) using
    the [resource group template](https://github.com/opensciencegrid/topology/blob/master/template-resourcegroup.yaml)
    as a guide.
-   If you are modifying existing entries, make sure you do not change formatting or indentation of the modified entry.
```

Comment:
Please keep this line.

Reply:
We need to warn people of the dangers of significant whitespace.

Reply:
Good point, I'll reword it so it's saner